playwright/tests/inscription/mock/browser/conditionBuilder.spec.ts

```diff
@@ -0,0 +1,61 @@
+import type { Page } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+import { InscriptionView } from '../../../page-objects/inscription/InscriptionView';
+import { browserBtn } from './browser-mock-utils';
+
+test.describe('Condition Builder', () => {
+  test('Add Condition', async ({ page }) => {
+    const inscriptionView = await InscriptionView.mock(page, { type: 'Alternative' });
+    const condition = inscriptionView.accordion('Condition');
+    await condition.toggle();
+    const conditionSection = condition.section('Condition');
+    await conditionSection.expectIsOpen();
+    const conditionTable = condition.table(['label', 'expression']);
+    const conditionCell = conditionTable.row(1).column(1);
+    await conditionCell.focusScriptCell();
+    await browserBtn(page).nth(0).click();
+    await applyConditionBuilder(page);
+    await conditionCell.expectValue('(data.value1 == 10 && data.value2 > 5) || (data.value3 < 6)');
+  });
+});
+
+export async function applyConditionBuilder(page: Page) {
+  const browserDialog = page.getByRole('dialog');
+  await expect(browserDialog).toBeVisible();
+  await browserDialog.getByText('Condition').first().click();
+  const condition = browserDialog.locator('.ui-condition-builder-condition');
```

Comment:
I guess I could theoretically create a condition builder page object, but I'm not sure if it's worth the effort since I only need it for a single 'happy path' Playwright test in this case.

Reply:
I think I would be also fine if you leave this code in the test directly.
As long as there is no second test which needs the same logic, it is also fine if all is in the test itself

Reply:
Alright, makes sense 👍 but I think I'll stick with the applyConditionBuilder function since it follows the same pattern of the other browser-playwright tests.